website/content/docs/discovery/dns.mdx

```diff
@@ -353,23 +353,46 @@ $ echo -n "20010db800010002cafe000000001337" | perl -ne 'printf join(":", unpack
 
 By default, all service lookups use the `default` namespace
 within the partition and datacenter of the Consul agent that received the DNS query.
-
-Use the following query format to specify a namespace, partition, and/or datacenter
-for all service lookup types except `.query`,
-including `.service`, `.connect`, `.virtual`, and `.ingress`.
-At least two of those three fields (`namespace`, `partition`, `datacenter`)
-must be specified.
-```text
-[<tag>.]<service>.service.<namespace>.ns.<partition>.ap.<datacenter>.dc.<domain>
-```
+To lookup services in another namespace, partition, and/or datacenter,
+use the [canonical format](#canonical-format).
 
 Consul server agents are in the `default` partition.
 If DNS queries are addressed to Consul server agents,
 service lookups to non-`default` partitions must explicitly specify
 the partition of the target service.
 
 To lookup services imported from a cluster peer,
-use a [service virtual IP lookups for Consul Enterprise](#service-virtual-ip-lookups-for-consul-enterprise) instead.
+refer to [service virtual IP lookups for Consul Enterprise](#service-virtual-ip-lookups-for-consul-enterprise) instead.
+
+#### Canonical format
+
+Use the following query format to specify namespace, partition, and/or datacenter
+for all service lookup types except `.query`,
+including `.service`, `.connect`, `.virtual`, and `.ingress`.
+All three fields (`namespace`, `partition`, `datacenter`) are optional.
+```text
+[<tag>.]<service>.service[.<namespace>.ns][.<partition>.ap][.<datacenter>.dc]<domain>
+```
+
```

Comment:
```suggestion
Use the [prepared query lookup format](#prepared-query-lookups) for `.query` lookups. 

```
Correct?

Reply:
I like the suggestion. It's best to redirect people to where they need to go.

Unfortunately, my understanding is that prepared queries don't support enterprise locality labels at all. And if that's true, maybe that's what needs to be stated explicitly.

Reply:
I'll avoid making this change in this PR.

I'll make it in a follow-up PR because it requires some updates to the prepared query docs as well, and I don't want to delay the merging of the enterprise code PR for that follow-up PR.